### ISSUE TITLE
Leverage window --grid instead of size and pos

### ISSUE THREAD

**Issue by Bandgren** [arpandaze/scratchpad-rs#5]
Hi! 

Im liking this implementation and want to use it since im not able to disable SIP to leverage yabais own scratchpad.
Would you be open if i made a PR about moving from size/pos into using the grid command instead?
For example:
https://github.com/koekeishiya/yabai/wiki/Commands
```
# move focused window to occupy the left two thirds of the screen. 
yabai -m window --grid 1:3:0:0:2:1
```
This would help me a lot since i move contantly from laptop to 1 monitor or multiple monitors depending if i work from home/office or just on my laptop since the --grid command puts it evenly on all monitor resolutions!

I have made some super simple changes to make it work for me now locally, but instead of forking etc i would love to contribute if you are open into moving to the grid command instead!

**Comment by arpandaze** [comment]
Instead of entirely switching to grids, we could make the position and size fields optional in the configuration. We can then introduce a new optional field for the grid. If the position and size fields are not specified, the configuration will default to using the grid settings. This approach maintains compatibility with existing configurations. What do you think?

**Comment by Bandgren** [comment]
Hi! Yea that would probably be the absolute best tbh! Sadly I have extreamly limited knowledge of rust. :( 
Is that something rather easy for you to put in?

**Comment by arpandaze** [comment]
Shouldn't be too hard but I am little busy right now. I will let you know once I have implemented that.

**Comment by arpandaze** [comment]
Implemented on 1c677003d4f9cbcfd57b7b432a201ad35eabee37

Example:
```
[[scratchpad]]
name = "telegram"
target_type = "app"
target = "Telegram"
position = [1, 1]
size = [4, 4]
rows = 6
cols = 6
launch_type = "app"
launch_command = "Telegram.app"
```
